packages/scenario/vitest.config.ts

```diff
@@ -77,22 +85,27 @@ export default defineConfig(({ mode }) => {
 			exclude,
 
 			deps: {
-				optimizer: {
```

Comment:
Adding or removing this doesn't seem to give any improvement during test execution